101-vm-sql-full/azuredeploy.json

```diff
@@ -0,0 +1,590 @@
+{
```

Comment:
This is probably a 301 sql full template and maybe put all the scenarios in the name